src/unix/fs.c

```diff
@@ -901,6 +905,41 @@ static ssize_t uv__fs_sendfile_emul(uv_fs_t* req) {
 }
 
 
+#ifdef __linux__
+static unsigned uv__kernel_version(void) {
+  struct utsname u;
+  unsigned major;
+  unsigned minor;
+  unsigned patch;
+
+  if (-1 == uname(&u))
+    return 0;
+
+  if (3 != sscanf(u.release, "%u.%u.%u", &major, &minor, &patch))
+    return 0;
+
+  return major * 65536 + minor * 256 + patch;
+}
```

Comment:
Would it make sense caching the kernel version in a static variable?

Reply:
I don't think it can hurt - updated.